apps/game_backend/lib/game_backend/items.ex

```diff
@@ -244,4 +244,35 @@ defmodule GameBackend.Items do
       {:error, :character_cannot_equip}
     end
   end
+
+  @doc """
+  Gets Item's purchase cost by given currency and item_template.
+  Returns {:ok, purchase_cost} if found one.
+  Returns {:error, :not_found} if there are none.
+  Fails if there are more than one.
```

Comment:
This operation doesn't fail if there's more than one currency with that ID we return the first occurrence with Enum.find/2

Reply:
Done!